python/lsst/ts/standardscripts/auxtel/prepare_for/co2_cleanup.py

```diff
@@ -0,0 +1,141 @@
+            $id: https://github.com/lsst-ts/ts_standardscripts/auxtel/prepare_for/co2_cleanup.yaml
+            title: PrepareForCO2Cleanup v1
+            description: Configuration for PrepareForCO2Cleanup.
+            type: object
+            properties:
+                az:
+                    description: Target Azimuth in degrees.
+                    type: number
+                    default: 0.0
+                el:
+                    description: Target Elevation in degrees.
+                    type: number
+                    minimum: 0.0
+                    maximum: 90.0
+                    default: 20.0
+                rot_tel:
+                    description: Rotator angle in mount physical coordinates (degrees).
+                    type: number
+                    default: 0.0
+                target_name:
+                    description: Name of the position.
+                    type: string
+                    default: "CO2 cleanup position"
+                wait_dome:
+                    description: Wait for dome to be in sync with the telescope?
+                    type: boolean
+                    default: false
+                slew_timeout:
+                    description: Timeout for slew procedure (in seconds).
+                    type: number
+                    default: 180.0
+                ignore:
+                    description: ATCS components to ignore in availability check.
+                    type: array
+                    items:
+                        type: string
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+        self.config = config
+
+        if self.atcs is None:
+            self.atcs = ATCS(
+                domain=self.domain, intended_usage=ATCSUsages.Slew, log=self.log
+            )
+            await self.atcs.start_task
+
+        if hasattr(self.config, "ignore"):
+            for comp in self.config.ignore:
+                if comp not in self.atcs.components_attr:
+                    self.log.warning(
+                        f"Component {comp} not in ATCS. "
+                        f"Must be one of {self.atcs.components_attr}. Ignoring."
+                    )
+                else:
+                    self.log.debug(f"Ignoring component {comp}.")
+                    setattr(self.atcs.check, comp, False)
+
+    def set_metadata(self, metadata):
+        metadata.duration = self.slew_time_guess
+
+    async def run(self):
+        await self.atcs.assert_all_enabled()
+
+        self.log.debug(
+            "Slew telescope to CO2 cleanup position. "
+            f"Az: {self.config.az}, "
+            f"El: {self.config.el}, "
+            f"Rot: {self.config.rot_tel}"
+        )
```

Comment:
A couple things I forgot here, before moving the telescope, we need to enable ATAOS corrections. This will put pressure behind the mirror and make sure it is not moving on the states supports. you can enable it using

```
await atcs.enable_ataos_corrections()
```